src/ProjectService.Mappers/RequestsMappers/DbProjectMapper.cs

```diff
@@ -2,13 +2,14 @@
 using LT.DigitalOffice.ProjectService.Models.Db;
 using LT.DigitalOffice.ProjectService.Models.Dto.Requests;
 using System;
+using System.Collections.Generic;
 using System.Linq;
 
 namespace LT.DigitalOffice.ProjectService.Mappers.RequestsMappers
 {
     public class DbProjectMapper : IDbProjectMapper
     {
-        public DbProject Map(ProjectRequest request, Guid authorId)
+        public DbProject Map(ProjectRequest request, Guid authorId, List<Guid> users)
```

Comment:
You don't need it

Reply:
If you validate that all users exist you can use only requrst

Reply:
ok